packages/DropdownMenu/src/DropdownMenu.styles.js

```diff
@@ -5,6 +5,8 @@ import stylers from "@paprika/stylers";
 export const contentStyles = css`
   margin-left: -${tokens.spaceLg};
   margin-right: -${tokens.spaceLg};
+  overflow-x: hidden;
```

Comment:
not sure if this is necessary, did you find a particular situation that you needed it?

Reply:
I'm seeing a bug (on master as well, so perhaps not related but perhaps fixing in this pr when trying to focus the edit option in the dropdown via keyboard navigation...
![image](https://user-images.githubusercontent.com/4040102/83201223-e2c3b900-a0f9-11ea-898c-0e831f8819cb.png)
If not fixed here perhaps create another ticket (or let me know and i can create one) so that we are tracking that issue

Reply:
For the `overflow-x:hidden`, if the menu item uses a long word it overflows. 
![overflowx](https://user-images.githubusercontent.com/11863935/83431481-472d9380-a3ec-11ea-822c-d43b637d7363.gif)

There's a `max-height` for the `<DropdownMenu/>` so I'm not sure how to approach it. Either use `overflow-x:hidden`
![overflowxHidden](https://user-images.githubusercontent.com/11863935/83431163-c078b680-a3eb-11ea-89fb-3a52b59368c5.gif)

or `overflow-wrap:break-word`
![overflowWrap](https://user-images.githubusercontent.com/11863935/83431172-c40c3d80-a3eb-11ea-8a86-b053900bb1ed.gif)

Reply:
Let's remember its a menu, no really a content holder, I think to apply overflow: hidden should be good for now.

Reply:
Agree, go with `overflow: hidden` for now as I imagine could run into issues if there is a max-height on the dropdown.... We can always revisit this issue